docs/migrate.rst

```diff
@@ -1,6 +1,45 @@
 Migration Guide
 ===============
 
+Migrating to Rally 1.1.0
+------------------------
+
+``request-params`` in operations are passed as is and not serialized
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+With Rally 1.1.0 any operations supporting the option ``request-params`` property will pass the structure as is without attempting to serialize values.
```

Comment:
Remove "option"? (or did you mean to say "option**al**"?)

Reply:
Precisely, it was a typo. Fixed in a0fe4763761593a9ca29defcfeb073ba9d2a54a8